modules/xmpp/moderator.js

```diff
@@ -102,7 +102,6 @@ Moderator.prototype.isSipGatewayEnabled = function() {
 };
 
 Moderator.prototype.onMucMemberLeft = function(jid) {
-    logger.info(`Someone left is it focus ? ${jid}`);
```

Comment:
Hum, we do not have a log for someone leaving the room if we remove this one ...

Reply:
I see plenty of them. You mean in LJM itself?

Reply:
I meant in general ... but yeah, you are right `USER 6c59086d LEFT` from jitsi-meet, sorry